README.md

```diff
@@ -35,6 +35,13 @@ This restriction makes the Prometheus pushgateway inappropriate for the usecase
 
 See https://github.com/weaveworks/promjs/ for a JS client library for Prometheus that can be used from within a web app.
 
+## Getting Started
+Mostly, registering metrics and pushing are the same as the official push gateway
```

Comment:
Can you add a link here?  Not everyone will arrive here via the same path you did.